bigframes/core/__init__.py

```diff
@@ -94,6 +95,7 @@ def __init__(
         hidden_ordering_columns: Optional[Sequence[ibis_types.Value]] = None,
         ordering: ExpressionOrdering = ExpressionOrdering(),
         predicates: Optional[Collection[ibis_types.BooleanValue]] = None,
+        api_methods: Dict[str, str] = {},
```

Comment:
Why is this a dictionary? Don't we just need a list?

Reply:
Change to list done.